README.md

```diff
@@ -95,12 +95,12 @@ const StyleDictionary = require('style-dictionary').extend({
   source: ['tokens/**/*.json'],
   platforms: {
     scss: {
-      transformGroup: 'scss',
+      transformGroup: predefinedTransformGroupNames.scss,
```

Comment:
I guess the cleanest API for our examples would be:

```js
import { transformGroups } from 'style-dictionary/enums';

const { scss } = transformGroups;

{
  transformGroup: scss
}

Reply:
I agree this would be very easy to read. Just keep in mind the overlap between `predefinedTransformGroupNames` and `predefinedPropertyFormatNames`. Partly the same string values (`css`, `less`) have been used for different properties.

Reply:
I think that's fine, the string values can overlap among multiple enums, even within the same enum's members, but it allows us in the future (> v5, right now the values are public API) to change the values without it being breaking changes if the public API is only designated as the enum keys. Similarly, package entrypoints keys are public APIs whereas its values are private, meaning the locations of files corresponding to the entrypoints can be changed at any time by the maintainer, without that being breaking.